components/layout/Error.js

```diff
@@ -7,8 +7,9 @@ const defaultMessages = {
     500: 'Internal Server Error'
 };
 
-function Error({ statusCode, message }) {
+function Error({ statusCode, message, stack }) {
     const displayMessage = message || defaultMessages[statusCode];
+    console.error(stack)
```

Comment:
I think we should use the logger here: https://github.com/dotCMS/dotcms-spa/blob/issue-17617-better-error-handling/utils/logger.js#L1

We might want to extend the logger.